isis/src/base/apps/overlapstats/overlapstats.cpp

```diff
@@ -0,0 +1,314 @@
+#include <sstream>
+#include <set>
+
+#include "overlapstats.h"
+#include "SerialNumberList.h"
+#include "ImageOverlapSet.h"
+#include "ImageOverlap.h"
+#include "Statistics.h"
+#include "PvlGroup.h"
+#include "Pvl.h"
+#include "Target.h"
+#include "TProjection.h"
+#include "ProjectionFactory.h"
+#include "PolygonTools.h"
+#include "Progress.h"
+
+using namespace std;
+using namespace Isis;
```

Comment:
This is kinda everywhere and we should probably remove it. No need for using namespace if everything is in the same namespace.